### ISSUE TITLE
No Images Visible in BM Manuscript Records

### ISSUE THREAD

**Issue by Ralph-Lee-UK** [BetaMasaheft/Manuscripts#1849]
Images don't seem to be visible for any of the records that we have encoded. Can you help?

**Comment by eu-genia** [comment]
sure, should be all right now

**Comment by eu-genia** [comment]
Which link exactly do you mean? Did it work before? Can you put a snapshot?
I can see the photos under Images and this is the only link I am aware of...

**Comment by Ralph-Lee-UK** [comment]
For istance, on this record, clicking on the 'Fols. 1r-88r' link would have brought up a new window with the first image of that range

https://betamasaheft.eu/manuscripts/EMIP01950/main#1r

![image](https://user-images.githubusercontent.com/39619238/189080934-b9e1261e-4ef7-4b67-a713-477cdab81f45.png)

**Comment by Ralph-Lee-UK** [comment]
Yes it was. But I'm not sure when it was last working as I haven't looked for some time

**Comment by eu-genia** [comment]
Thank you. I cannot promise you that I can solve this fast but I will try to find out why it stopped working.
